lib/controller/controller.go

```diff
@@ -1085,3 +1085,12 @@ func (ctrl *ProvisionController) getStorageClassFields(name string) (string, map
 func claimToClaimKey(claim *v1.PersistentVolumeClaim) string {
 	return fmt.Sprintf("%s/%s", claim.Namespace, claim.Name)
 }
+
```

Comment:
@wongma7 I agree. Do you want to wait for util functions to create one. Otherwise I can do it with this PR . Please let me know what do you think ?